lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -92,13 +89,20 @@ export default function LMSFilePicker({
   // Fetches files or shows a prompt to authorize access.
   const fetchFiles = useCallback(async () => {
     try {
-      setDialogState({ ...INITIAL_DIALOG_STATE, state: 'fetching' });
+      // Set `continueAction` at the end of this function, depending of
+      // the outcome of the fetch. This prevents short re-renderings of the button,
+      // as seen mentioned here: https://github.com/hypothesis/lms/pull/2219#issuecomment-721833947
```

Comment:
I found it difficult to relate the comment to the change here. My understanding is that what this code is actually doing is preserving the `continueAction` from the previous state instead of resetting it as we did before. Is that correct?

If so I would suggest to change the comment to something like "Show the fetching state, but preserve the existing `continueAction` to prevent the button label changing. See https://github.com/hypothesis/lms/pull/2219#issuecomment-721833947".

Reply:
It doesn't look to me like the comment has been updated. Do you have local changes that haven't been pushed?